README.md

```diff
@@ -1,2 +1,234 @@
-# module-ballerinax-sap.jco
-The Ballerina SAP JCo Connector enables seamless integration with SAP systems using the SAP Java Connector (JCo), facilitating efficient and reliable enterprise-grade integrations.
+# Ballerina SAP Connector
```

Comment:
Is this title correct? I just see a mismatch in this and repo. Ignore if it is correct.

Reply:
Shall we rename it "SAP JCo Connector" @niveathika, or do we have other recommendations?